Makefile.am

```diff
@@ -74,6 +75,7 @@ TOR_UTIL_TESTING_LIBS = \
         src/lib/libtor-geoip-testing.a \
 	src/lib/libtor-process-testing.a \
         src/lib/libtor-buf-testing.a \
```

Comment:
You might not see it in github's rendering, but this line and the libto-geoip line above use spaces instead of tabs. Dunno if you want to fix it here or as a separate commit or ticket.

Reply:
Added #30991 for an automatic fix; I think if we don't automate this, I will continue messing it up in the future.